MailerQ/OutgoingMessage.cs

```diff
@@ -10,8 +10,20 @@ namespace MailerQ
         NamingStrategyType = typeof(LowercaseNamingStrategy),
         ItemNullValueHandling = NullValueHandling.Ignore
     )]
-    public class OutgoingMessage : Message
+    public class OutgoingMessage
     {
+        /// <summary>
+        /// This is a custom property to allow follow the message while the delivery process
+        /// </summary>
+        [Obsolete("Use the MessageId property instead that is currently used as its backend property. This will be removed in the next major version.")]
+        public string DeliveryGuid { get => MessageId; set => MessageId = value; }
+
+        /// <summary>
+        /// Unique message id generated for the mail
+        /// </summary>
+        [JsonProperty("message-id")]
+        public string MessageId { get; set; }
```

Comment:
If I am not wrong, at least in Relay, _Delivery ID_ is not the same as _Message ID_ because, when you send a message with many copies, all of them have the same _Message ID_ but each of them has his own _Delivery ID_.

I am out of context, so, maybe I am understanding it wrong, but maybe it is a good excuse for me, to learn a little more about this.